scripts/tests/test_migrate_profile_websites.py

```diff
@@ -0,0 +1,69 @@
+from tests.factories import AuthUserFactory
+from nose.tools import *
+from tests.base import OsfTestCase
+from scripts.migration.migrate_personal_to_profile_websites import main, get_users_with_social_field
+
+
+class TestMigrateProfileWebsites(OsfTestCase):
+
+    def setUp(self):
+        super(TestMigrateProfileWebsites, self).setUp()
+        self.user_one = AuthUserFactory.build(
```

Comment:
This is fine, but for future reference, you can use `UserFactory` rather than `AuthUserFactory` when you don't need to do HTTP Basic Auth in your tests (e.g .`auth=user.auth`). This makes tests faster because UserFactory doesn't need to set the user's password.